### ISSUE TITLE
use gh-pages branch instead of web folder

### ISSUE THREAD

**Issue by rom1504** [extremeheat/minecraft-inventory-gui#11]
https://github.com/js-org/js.org/pull/6747#issuecomment-1002253844

**Comment by extremeheat** [comment]
Hmm, ok. Not sure what the best way to go about this but we could add some bundling step and build script to deploy to gh-pages, or just move the web/ folder to the root of gh-pages branch and update the import paths

**Comment by rom1504** [comment]
yeah having some bundling step and publishing to gh pages automatically would be ideal
a bit like https://github.com/PrismarineJS/prismarine-viewer/blob/master/.github/workflows/ci.yml

**Comment by rom1504** [comment]
that might help improve the packaging of the lib as well / make a better example
